src/dmd/dtoh.d

```diff
@@ -671,34 +744,32 @@ public:
             if (adparent)
                 buf.writestring("static ");
             typeToBuffer(vd.type, vd.ident);
-            buf.writestring(";\n");
-            if (!adparent)
-                buf.printf("\n");
+            writeDeclEnd();
             return;
         }
 
         if (adparent && vd.type && vd.type.deco)
         {
-            indent();
             auto save = cdparent;
             cdparent = vd.isField() ? adparent.isClassDeclaration() : null;
             typeToBuffer(vd.type, vd.ident);
             cdparent = save;
-            buf.writestring(";\n");
+            buf.writestringln(";");
 
             if (auto t = vd.type.isTypeStruct())
                 includeSymbol(t.sym);
 
-            auto savex = buf;
-            buf = checkbuf;
-            buf.writestring("    assert(offsetof(");
-            buf.writestring(adparent.ident.toChars());
-            buf.writestring(", ");
-            buf.writestring(vd.ident.toChars());
-            buf.printf(") == %d);\n", vd.offset);
-            buf = savex;
+            checkbuf.level++;
+            const pn = adparent.ident.toChars();
+            const vn = vd.ident.toChars();
+            const vo = vd.offset;
+            checkbuf.printf("assert(offsetof(%s, %s) == %d);",
+                                             pn, vn,    vo);
```

Comment:
I see why you'd do that but it break auto-indentation for most people

Reply:
I don't think the line will be touched.

Reply:
Well it is at the moment, and the style you are using has no precedent in DMD.